prometheus/prometheus.yml

```diff
@@ -10,4 +10,4 @@ scrape_configs:
 - job_name: scylla
   honor_labels: true
   static_configs:
-  - targets: ['127.0.0.1.18:9103']
+  - targets: ['127.0.0.1:9090']
```

Comment:
I referenced the example in prometheus github [1]. I will keep using 9103 port when I rebase the PR. 

[1] https://github.com/prometheus/prometheus/blob/master/documentation/examples/prometheus.yml